palantirnet/st-bernards
# MSSB-56: Image style changes and additions.

## JIRA Ticket(s)

- See ticket https://palantir.atlassian.net/browse/MSSB-56

## Description

- Installed Image Styles Mapping module (not committed) to analyze the image styles being used on the site
![Screen Shot 2020-01-21 at 11 21 42 AM](https://user-images.githubusercontent.com/10488517/72831790-2ccaf880-3c49-11ea-9310-4c5d4b9c5c9c.png)
![Screen Shot 2020-01-21 at 11 21 52 AM](https://user-images.githubusercontent.com/10488517/72831802-2fc5e900-3c49-11ea-8a2e-5bf31a44239d.png)

- Went through the images that have no image styles set, created image styles to be used for the image fields in the future
- Location logo - use Location Logo image style
- Location Image - used in Location view listing page (http://st-bernards.local/locations). Use Location View Image image style
- News Featured Image - used in News page (http://st-bernards.local/news/blog-antibiotic-isnt-always-answer). Use News Page Image image style
- Provider Image - used in Provider view listing page (http://st-bernards.local/providers) and Provider profile page. Use Provider Image image style in both places

## Testing instructions

- `drush cim`
- `drush cr`
- Visit the image style page and check that the new image styles have been created. (http://st-bernards.local/admin/config/media/image-styles)
